### ISSUE TITLE
update /etc/kubernetes/kubelet.conf when upgrade

### ISSUE THREAD

**Issue by hlwanghl** [QingCloudAppcenter/QKE#294]
the original content of the file for QKE v2 is:
```
users:
- user:
    client-certificate-data: xxx...
    client-key-data: yyy...
```

it will fail to use renewed certs. we need to change it to:
```
users:
- user:
    client-certificate: /var/lib/kubelet/pki/kubelet-client-current.pem
    client-key: /var/lib/kubelet/pki/kubelet-client-current.pem
```

so it will always points to the latest certs.

**Comment by hlwanghl** [comment]
only need to update the file of master1.

**Comment by hlwanghl** [comment]
it's caused by kubeadm bug: https://github.com/kubernetes/kubeadm/issues/1753

```
kubeadm init phase kubelet-finalize experimental-cert-rotation
```

**Comment by hlwanghl** [comment]
Fixed by #316
